modules/aerodyn/src/BEMT.f90

```diff
@@ -636,7 +636,7 @@ subroutine BEMT_Init( InitInp, u, p, x, xd, z, OtherState, AFInfo, y, misc, Inte
    if (errStat >= AbortErrLev) return
 
    InitInp_DBEMT%DBEMT_Mod = p%DBEMT_Mod
-   if ( p%DBEMT_Mod > DBEMT_none ) then
+   if ( p%DBEMT_Mod > DBEMT_none .or. p%DBEMT_Mod == DBEMT_Frozen  ) then
```

Comment:
Actually... Are you sure the DBEMT module is _supposed_ to be called with frozen wake? I thought it just called a routine in BEMT instead of the DBEMT module.

Reply:
I'm thinking the places where we check `p%DBEMT_Mod /= DBEMT_none` should be changed to `p%DBEMT_Mod > DBEMT_none` to avoid the seg fault you were seeing.

Reply:
It seems we need `DBEMT_Init` to get called, otherwise we get a seg fault.  But perhaps I addressed the wrong issue with that change.

Reply:
I did some quick testing.  @bjonkman, I think you are correct. Changing some of the `p%DBEMT_Mod /= DBEMT_none` to `p%DBEMT_Mod > DBEMT_none` instead fixes the segfault.  I'll post a PR later this evening to revert commit 7305adb67 and add this correction.